src/cli.ts

```diff
@@ -91,8 +91,26 @@ export const semverToInt = (input: string) =>
 export const camelToHyphen = (str: string) =>
 	str.replace(/([A-Za-z])(?=[A-Z])/g, "$1-").toLowerCase();
 
-export const sanitizeInput = (str: string) =>
-	str.replace(/(["$'\\`])/g, "\\$1");
+const specialCharacters = ['"', "$", "'", "\\", "`"];
```

Comment:
Couldn't this also be a Set, for consistency and a slight performance improvement?